runtime/js/unix.js

```diff
@@ -77,6 +91,22 @@ function caml_unix_startup() {}
 //Alias: win_cleanup
 function caml_unix_cleanup() {}
 
+//Provides: caml_sys_putenv (const, const)
+//Requires: caml_jsstring_of_string, jsoo_static_env
+function caml_sys_putenv(name, value) {
+  var name = caml_jsstring_of_string(name);
+  var value = caml_jsstring_of_string(value);
+  delete jsoo_static_env[name];
```

Comment:
getenv can be optimized at compile time based on the value at static env variable. I'm not sure we should alter it  at runtime.

Reply:
I have removed `putenv`. It is not very useful anyway.